src/subscription_manager/cli_command/abstract_syspurpose.py

```diff
@@ -187,10 +187,8 @@ def _is_provided_value_valid(self, value):
                     self.options.token:
                 if self.attr in valid_fields:
                     if len(valid_fields[self.attr]) > 0:
-                        print(_('Warning: Provided value "{val}" is not included in the list of valid values').format(
-                            val=value
-                        ))
-                        self._print_valid_values(valid_fields)
```

Comment:
ok, so what exactly do I need to do here?

Reply:
Here, nothing. The actual changes should be done few lines above:
```python
        # First check if the the value is in the valid_fields
        ret = False
        valid_fields = self._get_valid_fields()
        if self.attr in valid_fields:
            if value in valid_fields[self.attr]:
                ret = True
```
In particular, the search should be done with case-insensitive comparisons, rather than exact string matching.

Reply:
ok, that is what I was thinking, but wanted to be sure.  Thank you - I shall make it so and add a test :)

Reply:
and thank you for getting back with me!